packages/@okta/vuepress-site/docs/guides/enable-express-configuration/main/index.md

```diff
@@ -0,0 +1,300 @@
+
+```bash
+auth0 api patch tenants/settings \
+  --data '{
+    "flags": {
+      "use_scope_descriptions_for_consent": true
+    }
+  }'
+```
+
+## Step 7: Email the Okta Express Configuration team
+
+Email following information to the Okta Express Configuration team at [expressconfig@okta.com](mailto:expressconfig@okta.com):
+
+1. Confirmation that you completed all the steps in this guide and your app is ready to support Express Configuration.
+2. Your app name in the OIN.
+3. The OIN Integration Client ID.
+
+The Okta Express Configuration team configures your app in the OIN and assigns it to your developer org. You can then test the feature by creating an instance of your app in the OIN catalog.
+
+# Verification and testing
+
+Follow these steps to verify and test the Express Configuration feature:
+
+1. Sign in to your [Okta Developer Edition org](/login/) as a user with either the super admin (`SUPER_ADMIN`) role, or the app (`APP_ADMIN`) and org (`ORG_ADMIN`) admin [roles](https://developer.okta.com/docs/api/openapi/okta-management/guides/roles/#standard-roles).
+1. Go to **Applications** > **Your OIN Integrations** in the Admin Console.
+1. Click **Browse App Catalog** and search for your app.
+1. Open your app's detail page and click **Add Integration**.
+1. In **General Settings**, click **Done** to create an instance of your OIN app.
+1. Go to the **Authentication** tab.
+1. Click **Configure SSO with OIDC** and complete the authorization flow.
+
+<div class="half wireframe-border">
+
+![Configure SSO with OIDC](/img/oin/ec_configure_SSO_with_OIDC.png "Configure SSO with OIDC")
+
+</div>
+
+8. Assign a test Okta user to this app instance.
+9. Sign in to your Okta Developer Edition org using this test user and click your app tab.
+10. Verify that the user is successfully signed in to your app.
+
+**Note**:
+When users use Express Configuration to set up SSO for an instance of your app in Okta, the following default configurations are applied to the newly created Okta Workforce Connection in Auth0. Users can’t modify these configurations:
+
+**Connection Settings**
+
+* **Scopes**: `openid email profile`
+* **User Mapping**: `{"mapping_mode" : "basic_profile"}`
+* **Connection Profile**: `{"pkce":"auto"}`
+
+**Login Experience**
+
+* **Home Realm Discovery**: Empty (not supported)
+
+* **Connection Button**
+    * **Display connection as a button**: Disabled (Enabled through **Organizations**)
+    * **Button Display Name**: Okta
+    * **Button Logo URL**: `https://cdn.brandfolder.io/R30ALRIS/at/scvv8tj7w545j3r5gmxq4jrb/Okta_Aura_Logomark_Black_RGB.png` (Okta brand logo)
+
+**Organizations**
+
+* **Membership On Authentication**: Enable **Auto-Membership**
+* **Display Connection as a button**: Enabled in login experience customization
```

Comment:
Login Exp and Org are all part of Connection settings. The structure shall look like

- Connection settings
    - scopes
    - user mapping
    - connection profile
    - login experience
        - HRD
    - Org
        - Member..
        - Display as button

Reply:
I looked at where all those config lives in Auth0, I think we shall go with
```
Connection settings
  - scopes
  - user mapping
  - connection profile
Connection login experience per Org
  - HRD: empty
  - Org auto Member : false
  - Display as button
```